iot/iot-tenant-service/src/main/java/io/enmasse/iot/tenant/impl/TenantServiceImpl.java

```diff
@@ -131,4 +129,16 @@ public void get(final X500Principal subjectDn, final Span span,
                 CacheDirective.maxAgeDirective(this.configuration.getCacheTimeToLive().getSeconds()));
     }
 
+    @Override
+    public void get(String tenantId, Handler<AsyncResult<TenantResult<JsonObject>>> resultHandler) {
+        // TODO Auto-generated method stub
```

Comment:
Are the empty methods intentional?  If so, I think it is worthy of a comment.

Reply:
Woohoo … no, they are not! Thanks for spotting this. Should be fixed now.